install/linux.md

```diff
@@ -81,9 +81,7 @@ Hier müssen im Terminal die folgenden Zeilen eingegeben werden:
 ```bash
 curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh"
 bash Mambaforge-$(uname)-$(uname -m).sh -p ~/.local/mambaforge -b
-source "${HOME}/.local/mambaforge/etc/profile.d/conda.sh"
 source "${HOME}/.local/mambaforge/etc/profile.d/mamba.sh"
-mamba activate
```

Comment:
With the new suggestions of restarting the terminal and using `init` instead of activate it still only works if I manually source both `conda` and `mamba` in the `~/.local/mambaforge/etc/profile.d/` directory.
Then I can `mamba activate` in that exact active session.

Also `$SHELL` and `$0` both return `/bin/zsh` now on my other system.

Reply:
What about `ps -ocomm= -p $$` ?

Reply:
> With the new suggestions of restarting the terminal and using init after activate it still only works if I manually source both conda and mamba in the ~/.local/mambaforge/etc/profile.d/ directory.

The new suggestion is *not* to do `activate`.

With the changed shell part, the instructions should be:

```
$ bash Mambaforge-$(uname)-$(uname -m).sh -p ~/.local/mambaforge -b
$ source "${HOME}/.local/mambaforge/etc/profile.d/mamba.sh"
$ mamba init $(ps -ocom= -p $$)
```
Then close current and open a fresh terminal.

Reply:
What about mamba init --all?

Reply:
> What about `ps -ocomm= -p $$` ?

That works and returns `zsh`

Reply:
> The new suggestion is _not_ to do `activate`.
> 
> With the changed shell part, the instructions should be:
> 
> ```
> $ bash Mambaforge-$(uname)-$(uname -m).sh -p ~/.local/mambaforge -b
> $ source "${HOME}/.local/mambaforge/etc/profile.d/mamba.sh"
> $ mamba init $(ps -ocom= -p $$)
> ```
> 
> Then close current and open a fresh terminal.

In that case I get
```
~ via  v20.3.0 ❯ mamba init $(ps -ocomm= -p $$)
__mamba_exe:3: no such file or directory: ./mamba
```

Reply:
> What about mamba init --all?

Here same, that no ./mamba exists.

It looks like the initial activate sorts these dependencies.

Also my `mamba.sh` looks like this.
```
mamba() {
    \local cmd="${1-__missing__}"
    case "$cmd" in
        activate|deactivate)
            __conda_activate "$@"
            ;;
        install|update|upgrade|remove|uninstall)
            __mamba_exe "$@" || \return
            __conda_reactivate
            ;;
        *)
            __mamba_exe "$@"
            ;;
    esac
}
```
which results in the _no mamba exe found_ error

Reply:
Are there any news or are we left with activating both conda and mamba manually before calling the init?